doc/script_commands.txt

```diff
@@ -7265,10 +7265,26 @@ one-on-one duel, for example.
 
 *kickwaitingroomall({"<NPC object name>"})
 
+	@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+	@ /!\ This command is deprecated @
+	@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
 This command kicks everybody out of a specified waiting room chat.
 
 ---------------------------------------
 
+*waitingroomkick({"<NPC object name>"{,"<player name>"|<account_id>}})
+
+This command kicks target player(s) out of a waiting room chat of specific npc if any.
+
+Examples:
+	waitingroomkick();
+	waitingroomkick("target_npc_name");
+	waitingroomkick("target_npc_name", "target_player_name");
+	waitingroomkick("target_npc_name", 200001);
```

Comment:
Bad Examples
How about use mine

```
Example 1:
//	This will kick all players in the waitingroom
	waitingroomkick();

Example 2:
//	Kick the last player enter the waitingroom
	.@num = getwaitingroomstate(0);
	waitingroomkick(strnpcinfo(NPC_NAME), $@chatmembers[.@num -1];
```